drivers/hmis/app/models/hmis/user.rb

```diff
@@ -22,11 +22,17 @@ class Hmis::User < ApplicationRecord
   has_many :access_controls, through: :user_groups
   has_many :access_groups, through: :access_controls
   has_many :roles, through: :access_controls
+  has_many :activity_logs, class_name: 'Hmis::ActivityLog'
 
   has_recent :clients, Hmis::Hud::Client
   has_recent :projects, Hmis::Hud::Project
   attr_accessor :hmis_data_source_id # stores the data_source_id of the currently logged in HMIS
 
+  scope :hmis_users, -> do
```

Comment:
If I call `Hmis::User.hmis_users` it's a little hard to tell what it does just by the name. Perhaps `Hmis::User.active_members` or "operational_users" would be more descriptive?

Reply:
Hmm yeah I agree. I want to capture that they are _hmis_ users -- warehouse-only users are not included. Maybe it would be better to split into two descriptive scopes like `Hmis::User.active.with_hmis_access`